bin/projects/dbatools/dbatools/Utility/DbaDate.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+
+namespace Sqlcollaborative.Dbatools
+{
+    namespace Utility
+    {
+        /// <summary>
+        /// A dbatools-internal datetime wrapper for neater display
+        /// </summary>
+        public class DbaDate : DbaDateTimeBase
+        {
+            #region Constructors
+            /// <summary>
+            /// Constructs a generic timestamp object wrapper from an input timestamp object.
+            /// </summary>
+            /// <param name="Timestamp">The timestamp to wrap</param>
+            public DbaDate(DateTime Timestamp)
+            {
+                _timestamp = Timestamp;
+            }
+
+            /// <summary>
```

Comment:
Yes, there is a reason.
Those classes were autogenerated from the original DateTime struct. I added the comments in order to shut Visual Studio up about them (I was willing to forego proper comments on a DateTime wrapper), so I'd see where I'd actual failed to provide proper help (because I consider those comments to be absolutely mandatory for original dbatools content).
Again a legacy of the old single-file project the old library mandated. I suppose they could be dropped now, but I don't see the reason since they'll flood the build log without added benefit that I can see. I'm fine either way though, so long as it doesn't mess with development.